samples/annotate/project-build.xml

```diff
@@ -159,10 +158,6 @@
 			<formatter type="xml"/>
 
 			<batchtest fork="yes" todir="${basedir}/target/junit-reports">
-				<fileset dir="${basedir}/src/test/java">
-					<include name="**/*Test*.java"/>
-					<exclude name="**/AllTests.java"/>
-				</fileset>
 				<fileset dir="${basedir}/target/generated-sources/xjc">
```

Comment:
maybe I could try to create those absent folders, instead of removing them from project-build.xml? Or just remove my mods in this file?